tensorboard/webapp/experiments/types.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+export interface Experiment {
+  // int64 should be represented in string as JS can only represent number up to
+  // 53 bits.
```

Comment:
Consider omitting this; TB services may provide different types of experiment id and string should cover all cases (and this detail does not matter).

Reply:
Done